CMakeLists.txt

```diff
@@ -105,7 +105,10 @@ find_package(Doxygen)
 find_package(GConf)
 find_package(Ldap)
 find_package(OpenDbx)
+
 find_package(PCRE REQUIRED)
+include_directories(${PCRE_INCLUDE_DIRS})
```

Comment:
I'd suggest adding it to line 551 instead.

Reply:
ok, I update the changeset